lib/config/plugins.js

```diff
@@ -117,6 +117,10 @@ module.exports = {
             } catch (err) {
                 debug("Failed to load plugin eslint-plugin-" + pluginNameWithoutPrefix + ". Proceeding without it.");
                 err.message = "Failed to load plugin " + pluginName + ": " + err.message;
+                err.messageTemplate = "plugin-missing";
+                err.messageData = {
+                    pluginName: pluginNameWithoutPrefix
+                };
```

Comment:
Basically, an error can now attach `messageTemplate` and `messageData` to automatically use a message template instead of throwing a regular stack trace errors.